apps/meeting/src/containers/MeetingProviderWrapper/index.tsx

```diff
@@ -17,6 +23,28 @@ import MeetingEventObserver from '../MeetingEventObserver';
 import meetingConfig from '../../meetingConfig';
 import { useAppState } from '../../providers/AppStateProvider';
 
+const MeetingProviderWithDeviceReplacement: React.FC = ({ children }) => {
+  const { addVoiceFocus } = useVoiceFocus();
+
+  const replaceDevice = (
+    nextDevice: string, 
+    currentDevice: Device | AudioTransformDevice
+  ): Promise<Device | VoiceFocusTransformDevice> => {
+    if (currentDevice instanceof VoiceFocusTransformDevice) {
+      return addVoiceFocus(nextDevice);
+    }
+    return Promise.resolve(nextDevice);
+  };
+
+  const meetingConfigValue = {
+    ...meetingConfig,
+    enableWebAudio: true,
+    onDeviceReplacement: replaceDevice
```

Comment:
Trailing comma.

If you call `replaceDevice` `onDeviceReplacement` you can simplify this line.

Reply:
Rename the function, it is simplified now :)